plugins/mcreator-core/procedures/entity_is_elytra_flying.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
As this procedure only applies to players, move it to Player procedures. Same with any other procedure that is the same case